gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskManager.java

```diff
@@ -63,8 +63,8 @@ public TaskManager(TaskUpdateNotifier taskUpdateNotifier,
     @Bean
     public Consumer<Flux<TaskStatusUpdate>> consumeTaskStatusUpdate() {
         return f -> f
-            .onErrorContinue((t, r) -> LOGGER.error(t.getMessage(), t))
-            .subscribe(this::handleTaskStatusUpdate);
+                .onErrorContinue((t, r) -> LOGGER.error(t.getMessage(), t))
+                .subscribe(this::handleTaskStatusUpdate);
```

Comment:
We have to harmonize that ! Initially with FARAO we were using only 4 spaces, we can stick to that or change but we have to all agree on that to not have these small diff all the time. It is configurable in IntelliJ :)